hdinsight-linux-with-hue-on-edge-node/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "itemDisplayName": "Create HDInsight Linux Cluster with Hue app on Edge Node",
   "description": "Template creates an HDInsight Linux cluster in a virtual network with another vm as an edge node that is bootstrapped with the clusters configurations and Hue is installed on edge node.",
```

Comment:
Let's change the description and summary. Template creates HDI cluster and installs Hue on an edgenode.